packages/fastify-renderer/test/routes.spec.ts

```diff
@@ -116,7 +116,8 @@ describe('FastifyRenderer', () => {
     expect(response.body).toEqual('hello')
   })
 
-  test('should call hooks in correct order', async () => {
+  // TODO: Re-introduce transformation hooks
```

Comment:
This test is tricky to re-write. Transform hook can't write to the call order stack.

Reply:
That checks out -- can we still test the same thing by rendering with the transforms, and then expecting the output to have stuff added by the hooks in the right order?

Reply:
Yea that's something we can do